src/dependency-version-checker.js

```diff
@@ -25,7 +27,7 @@ class DependencyVersionChecker {
     this.name = name;
     let addon = this._parent._addon;
     let basedir = addon.root || getProject(addon).root;
-    this._jsonPath = resolvePackage(this.name, basedir);
+    this._jsonPath = resolvePackagePath(this.name, basedir);
```

Comment:
Can you also move this down into the beginning of the getter for `version`? I think we can do this lazily too (that way just creating the version checker instance doesn't do any I/O, and we only would do I/O when `.version` was accessed).

Reply:
Good idea! I'll do that right now.